sysdig/resource_sysdig_secure_cloud_auth_account_test.go

```diff
@@ -755,6 +755,9 @@ resource "sysdig_secure_cloud_auth_account" "sample" {
 			api_key = {
 			  user_id = "user-id"
 			}
+#			policy = {
+#			 policy_id = "policy-id"
+#			}
 		  }
```

Comment:
You will also need to update the proto pb.go files once its merged.

Reply:
done